Widget.php

```diff
@@ -75,7 +75,7 @@ protected function registerClientOptions($name, $id)
     {
         if ($this->clientOptions !== false) {
             $options = empty($this->clientOptions) ? '' : Json::htmlEncode($this->clientOptions);
-            $js = "jQuery('#$id').$name($options);";
+            $js = "jQuery(document).on('ready pjax:success', function() { jQuery('#$id').$name($options); });";
```

Comment:
@samdark Thanks for the second set of eyes - would like your opinion on a few things.  First, to answer your question, the ready portion matches the current behavior, and the pjax:success adds the pjax support.
Taking a longer look at the code that's output, however, got me to thinking that the following may be a better approach?
[code]
$js = "jQuery(document).on('ready pjax:success', function() { jQuery('#$id').$name($options); });";
$view = $this->getView();
$view->registerJs($js, $view::POS_END);
[/code]
This would pull the JavaScript into a standalone end-of-body segment instead of adding it to the existing jquery(document).on('ready') segment.  Either way seems to be functionally equivalent, but this new snippet may be cleaner.
Let me know your thoughts and I can adjust as necessary.  Thanks!